tests/unit/inference/test_inference.py

```diff
@@ -337,6 +337,55 @@ def test(
         assert assert_fn(bs_output, ds_output)
 
 
+@pytest.mark.seq_inference
+@pytest.mark.parametrize("model_w_task",
+                         [("EleutherAI/gpt-j-6B",
```

Comment:
I think we should use a different model here. I recently moved our `gpt-j-6B` test to multi-GPU because we were often seeing OOM errors when running on a single GPU (see #2457 and #2493)